docs/ref/contrib/staticfiles.txt

```diff
@@ -228,6 +228,11 @@ Example usage::
 
     django-admin runserver --insecure
 
+.. note::
+
+  When serving static files, :djadmin:`runserver` delivers the files directly
+  and bypasses all installed middlewares.
```

Comment:
Sorry to be pedantic, but I'm not sure about using the plural "middlewares". The documentation isn't entirely consistent ("middlewares" does have a couple of uses), but the most common approach seems to be to treat "middleware" as a mass noun, i.e. not modifying it for number. For example, in `docs/ref/applications.txt`

    Applications include some combination of models, views, templates, template
    tags, static files, URLs, middleware, etc.

or in `docs/ref/request-response.txt`:

    Some of the middleware included in Django's contrib apps set attributes on the
    request.

Reply:
Agreed, and this plural issue has been mentioned before. 
https://code.djangoproject.com/ticket/27376#comment:3

I'll submit a separate issue to address the consistency of it across the docs.